### ISSUE TITLE
Error running server from example

### ISSUE THREAD

**Issue by tfalencar** [smartcontractkit/ccip-read#25]
I tried to give the example a spin, but failed when starting the server.

Node version: v16.15.0
OS: MacOS Monterey 12.5
Processor: Apple M1

cd ../server
yarn start

```
yarn run v1.22.17
$ yarn build && node dist/index.js
$ tsdx build
@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
✓ Creating entry file 2.2 secs
⠹ Building modules
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
(typescript) Error: /[redacted]/ccip-read/packages/examples/trusted-gateway-token/server/src/app.ts(1,24): semantic error TS2307: Cannot find module '@chainlink/ccip-read-server' or its corresponding type declarations.
Error: /[redacted]/ccip-read/packages/examples/trusted-gateway-token/server/src/app.ts(1,24): semantic error TS2307: Cannot find module '@chainlink/ccip-read-server' or its corresponding type declarations.
    at error (/[redacted]ccip-read/node_modules/rollup/dist/shared/node-entry.js:5400:30)
    at throwPluginError (/[redacted]/ccip-read/node_modules/rollup/dist/shared/node-entry.js:11878:12)
    at Object.error (/[redacted]/ccip-read/node_modules/rollup/dist/shared/node-entry.js:12912:24)
    at Object.error (/[redacted]/ccip-read/node_modules/rollup/dist/shared/node-entry.js:12081:38)
    at RollupContext.error (/[redacted]/ccip-read/node_modules/tsdx/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:17237:30)
    at /[redacted]/ccip-read/node_modules/tsdx/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:25033:23
    at arrayEach (/[redacted]/ccip-read/node_modules/tsdx/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:545:11)
    at Function.forEach (/[redacted]/ccip-read/node_modules/tsdx/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:9397:14)
    at printDiagnostics (/[redacted]/ccip-read/node_modules/tsdx/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:25006:12)
    at Object.transform (/[redacted]/ccip-read/node_modules/tsdx/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29277:17)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Note: Running `npx browserslist@latest --update-db` doesn't solve it.

**Comment by mdtanrikulu** [comment]
Hi! Could you retry installing dependencies and running again? What I see from logs is, `@chainlink/ccip-read-server` is not installed. 

Running `yarn` in `packages/examples/trusted-gateway-token/server` should help for the necessary dependencies.

**Comment by tfalencar** [comment]
I've tried that, but got the same error. I supposed running `yarn start` also invokes yarn install so it's effectively the same. It appears that setting the version like this solves the problem:
```
-    "@chainlink/ethers-ccip-read-provider": "0.2.2",
+    "@chainlink/ethers-ccip-read-provider": "^0.2.2",
```
After this was fixed, I've tried running the "client" part, and also hit problems in the import for:  "@chainlink/ethers-ccip-read-provider". 

This time adding "^" to the version of the package didn't help.

Then I realised that support for this should be already present in ethers, so I bumped to the latest version, and attempted to enable it by passing the "ccipReadEnabled: true" flag. Ethers seems to be doing the work of "ethers-ccip-read-provider", however when the following command is issued: 

TRANSFER 1 from 0x70997970C51812dc3A010C7d01b50e0d17dc79C8 to 0x8626f6940e2eb28930efb4cef49b2d1f2c9c1199

A few VM Exceptions are thrown, and execution is interrupted.

Since I might be already off-track from the original ticket, maybe we should close this one and I create a new ticket for the client ? In any case, should the 'client' be updated to make use of the latest ethers, instead of using ethers-ccip-read-provider ?
